handbook/engineering/security/goals.md

```diff
@@ -2,38 +2,51 @@
+8. Normalize log format in our centralized log storage so that it is easier to correlate and search. This is a prerequisite for creating automated alerts from the logs.
+9. Create alerts and dashboards to automate the process of investigating events of interest (e.g., detect and alert on a spike of failed login attempts to a single account, or across Sourcegraph Cloud as a whole).
+10. We run a time-bound capture the flag event where there are larger bounties for being able to gain access to our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+11. We connect our [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) to Sourcegraph Cloud and only [members who can access that repository on GitHub](https://github.com/sourcegraph/security-test/settings/access) can access that repository on Sourcegraph Cloud (i.e., [Sourcegraph organization owners](https://github.com/orgs/sourcegraph/people?query=role%3Aowner) and [@sourcegraph/security](https://github.com/orgs/sourcegraph/teams/security) members). We then advertise a bounty for each unique vulnerability that allows an unauthorized person to gain access to this [test security repository](https://github.com/sourcegraph/security-test/blob/main/README.md) on Sourcegraph Cloud.
+12. We host our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph Cloud.
+13. Document and publish our security practices so that our customers can 
+14. Document our roadmap and progress towards [SOC 3](#soc-audit-readiness).
+15. Validate that we are GDPR compliant.
+16. **Sourcegraph Cloud is a viable alternative to Sourcegraph for SMBs**
+
+
+
+### SOC Audit readiness
+
+**Problem:** As we begin providing a SaaS product, attaining a full SOC 2/3 certification will become necessary to enable the business. Note that a SOC 3 provides the same assurances as a SOC 2 certification, but contains less details, which makes it more accessible to general users, and something that we can [freely distribute](https://www.aicpa.org/interestareas/frc/assuranceadvisoryservices/aicpasoc3report.html). Therefore, we intend to pursue a SOC 3 certification in keeping with our company values of [openness and transparency](../../../company/values.md#open-and-transparent). If reasonable, we'll pursue SOC 2 type 2 in parallel with this, since it's the same audit process.
+
+1. Document our roadmap and progress towards SOC 3.
+1. Prepare for SOC 2 type 1 certification and contract with an auditor to complete the audit. Note that we begin with SOC 2 type 1 because type 1 audits can be achieved significantly faster than type 2 audits, and SOC 3 does not have a type 1 audit. This will also serve as a way to communicate to our customers that we're on the right track in our SOC certification process. Note that, unlike SOC 3, SOC 2 audits of both types cannot be freely distributed.
```

Comment:
Prepare for SOC 2 type 1 certification and contract with an auditor to complete the audit. Note that we begin with SOC 2 type 1 because type 1 audits can be achieved significantly faster than type 2 audits, and SOC 3 does not have a type 1 audit. This will also serve as a way to communicate to our customers that we're on the right track in our SOC certification process. SOC 3 is the focus as audit reports are designed to be more easily digestible than SOC 2 - nothing prevents us from sharing our SOC 2 audit artifacts.

Reply:
Updated WRT previous reworking of this block